scripts/DP27-JP/c100427002.lua

```diff
@@ -0,0 +1,107 @@
+--D・スキャナン
+--
+--Script by Trishula9
+function c100427002.initial_effect(c)
+	c:EnableReviveLimit()
+	--splimit
+	local e1=Effect.CreateEffect(c)
+	e1:SetType(EFFECT_TYPE_SINGLE)
+	e1:SetProperty(EFFECT_FLAG_CANNOT_DISABLE+EFFECT_FLAG_UNCOPYABLE)
+	e1:SetCode(EFFECT_SPSUMMON_CONDITION)
+	c:RegisterEffect(e1)
```

Comment:
I am pretty sure this piece of code indicates it cannot be Special Summoned by other ways, but this is not the case.
```suggestion
```